packages/kit/src/core/config/options.js

```diff
@@ -280,7 +280,9 @@ const options = object(
 			version: object({
 				name: string(Date.now().toString()),
 				pollInterval: number(0)
-			})
+			}),
+
+			fineGrainedSearchParamsInvalidation: boolean(false)
```

Comment:
I would put the TODO comment above here in the form of `// TODO v2: remove this option (always true)` and remove the comments in the other locations.